pkg/alerts/alertutils/alertutils.go

```diff
@@ -31,6 +31,7 @@ type AlertType uint8
 const (
 	AlertTypeLogs AlertType = iota + 1
 	AlertTypeMetrics
+	AlertTypeAPM
```

Comment:
never add a new enum type in the middle, it should be always at the end, otherwise it causes backwards compatability issue with existing data